### ISSUE TITLE
[shield] add things in resources/ (Trac #1158)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1152]
It would need some stuff in resources :

1.- resources/test is missing, according to http://software.icecube.wisc.edu/icerec_trunk/code_reviews.html it would need to include some python examples without command line parameters.

2.- resources/examples/dump_coincidence_data.py. If it is possible, try to remove from this example tableio, at some point examples should be minimal. 

3.- resources/doc/index.rst is missing, it needs some examples and explanation about what the project does to be included in http://software.icecube.wisc.edu/icerec_trunk/   

Migrated from https://code.icecube.wisc.edu/ticket/1158
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "It would need some stuff in resources :\n\n1.- resources/test is missing, according to http://software.icecube.wisc.edu/icerec_trunk/code_reviews.html it would need to include some python examples without command line parameters.\n\n2.- resources/examples/dump_coincidence_data.py. If it is possible, try to remove from this example tableio, at some point examples should be minimal. \n\n3.- resources/doc/index.rst is missing, it needs some examples and explanation about what the project does to be included in http://software.icecube.wisc.edu/icerec_trunk/   ", 
    "reporter": "gmaggi", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[shield] add things in resources/", 
    "priority": "major", 
    "keywords": "IceTop coincidence shower hypothesis", 
    "time": "2015-08-18T16:39:34", 
    "milestone": "", 
    "owner": "chris.weaver@icecube.wisc.edu", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150818T16:56:12: cweaver commented:

> 1. Is the request for tests of the python bindings or for example scripts? The former is reasonable and I can add some, but the latter does not make sense to me, as example code should be in resources/examples, not resources/test. 
> 
> 2. The purpose of this example is to demonstrate the use of this project with tableio. 
> 
> 3. The project description information can be copied from the existing documentation, although duplication seems suboptimal. Example code is already provided in resources/examples and copying that to the documentation does not make sense.

**Comment by icecube-trac** [comment]
Trac update at 20150818T17:22:17: gmaggi commented:

> 1.- I guess the main purpose of resources/test is to allow the user check how to run the module without making mistakes on passing arguments by command line, maybe, that would be what http://software.icecube.wisc.edu/icerec_trunk/code_reviews.html means. 
> I think the python binding would be good to include in resources/examples/dump_coincidence_data.py.
> 
> 2.- ok.
> 
> 3.- The information in index.dox could be copied to index.rst (if there is need to avoid duplication, probably split it in description and \param's, for example), maybe it would be good to put some description in index.rst about what it does and some examples how this runs  and the description of the parameters. I am aware about http://software.icecube.wisc.edu/icerec_trunk/inspect/shield.html, nevertheless, some information about this project in http://software.icecube.wisc.edu/icerec_trunk/ would be welcome, like what other projects in  icerec show.

**Comment by icecube-trac** [comment]
Trac update at 20150818T21:32:19: 
* cweaver changed status from "new" to "closed"
* cweaver commented:

> In changeset 136432:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136432"
> Bring documentation slightly more up-to-date.
> Add python bindings tests. 
> Duplicate docs so that something appears in sphinx. 
> Closes #1152.
> ```


* cweaver changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas commented:

> Milestone IceRec V5 deleted


* olivas changed milestone from "IceRec V5" to ""
